docs/app/css/style.css

```diff
@@ -307,7 +307,7 @@ code:not(.highlight) {
 .layout-content,
 .doc-content {
   max-width: 864px;
-  margin: auto;
+  margin: 0 16px;
```

Comment:
We probably don't want both margins and padding, there should be only 16px of space on the left and right.